### ISSUE TITLE
Navigation triangles positions erratic behavior

### ISSUE THREAD

**Issue by darrellschulte** [jnweaver/owl-slideshow-wp#7]
When viewing a post with a slideshow, the navigation triangles are centered on the image but when I reload, it creeps higher.

<img width="525" alt="triangles-middle" src="https://cloud.githubusercontent.com/assets/1561880/8836893/18dd3174-3089-11e5-812d-725afb238ed3.png">
<img width="513" alt="triangles-up" src="https://cloud.githubusercontent.com/assets/1561880/8836894/190baf90-3089-11e5-8679-9446f3a641c3.png">

**Comment by jnweaver** [comment]
Hmm? Not surprised that's buggy. I set the top position of those relative to the active image height via Javascript. I can't figure out a pure CSS way to do it since you don't know for sure how long a caption might be if there is one at all. So the height is variable. I try to calculate the image height and then reset top based on it. Do you have the above example somewhere where I can take a peek at it?

**Comment by darrellschulte** [comment]
No. It's just a local (VVV), stable install of WordPress 4.2.2 with zero plugins activated and the Twenty Fifteen theme.

**Comment by jnweaver** [comment]
What browser? (I should test on fresh install like that regardless.)

**Comment by darrellschulte** [comment]
Safari - Version 8.0.8 (10600.8.2)

I admit that I haven't tested on other browsers or other themes...

**Comment by darrellschulte** [comment]
Trying a new theme and testing on Safari, Chrome and Firefox.

Steps:
1. Loaded sample post with slideshow.
2. Reload page (Command-R).

Results on OS X:
- Safari, Version 8.0.8 (10600.8.2) - **navigation triangles move up as reported earlier.**
- Chrome, Version 44.0.2403.89 (64-bit) - navigation triangles remain located in original position.
- Firefox, 39.0 - navigation triangles remain located in original position.
